index.bs

```diff
@@ -4478,33 +4475,22 @@ entails in the ECMAScript language binding.
 <div class="example">
 
     Here is are some examples of wording to use to create and throw exceptions.
-    To throw a new [=simple exception=] named
-    {{TypeError}}:
+    To throw a new [=simple exception=] named <emu-val>TypeError</emu-val>:
 
     <blockquote>
-
-        [=Throw=] a TypeError.
-
+        [=Throw=] a <emu-val>TypeError</emu-val>.
```

Comment:
Oh :(. Not sure what to do then if we want to avoid breaking lots of specs. I guess we could add silly redirect definitions. Or try to get a few choice entries into Shepherd somehow.